challenges/arrayShift/array-shift.js

```diff
@@ -0,0 +1,16 @@
+
+const reverseArray = array => {
```

Comment:
here's another way to do the same thing. just for reference.

function reverse(a) {
	const newArr = [];
	for(let i = 0; i < a.length; i++) {
	newArr[i] = a[a.length - (i + 1)];
    }
return newArr;
}